kdwsdl2cpp/src/main.cpp

```diff
@@ -125,6 +129,13 @@ int main(int argc, char **argv)
             }
         } else if (opt == QLatin1String("-keep-unused-types")) {
             keepUnusedTypes = true;
+        } else if (opt == QLatin1String("-import-path")) {
+            ++arg;
+            if (!argv[arg]) {
+                showHelp(argv[0]);
+                return 1;
+            }
+            importPath = argv[arg];
```

Comment:
Shouldn't this be a list, so people want to specify -import-path multiple times, in case they have multiple local paths to search into? Alternatively, it shouldn't be called path but directory.
("path" makes me think of PATH and LD_LIBRARY_PATH etc. which are lists of directories)